### ISSUE TITLE
simple example - upload backup.sql

### ISSUE THREAD

**Issue by abubelinha** [davdtheemonk/PyRemoteSQL#1]
This looks like an interesting project but it lacks some documentation. Not sure if I understood what it does:
1. Is it supposed to connect to a mysql server through its phpmyadmin web interface? (if YES, that would be great) 
   Or is this simply for connecting to a mysql server in the usual direct way (NOT using a phpadmin web interface).

2. If above answer is YES, could you provide a couple of usage examples?
- how to run small queries (SELECT, INSERT, UPDATE, DELETE)
- is it possible to upload a big database dump file through phpmyadmin interface?

Thanks

**Comment by davdtheemonk** [comment]
Yeah it's supposed to connect via phpmyadmin and the direct way...but I had trouble writing the correct code to verify if phpmyadmin is installed in remote host..
1.There multiple OS's out there
2.Is there a way to verify Phpmyadmin is installed without causing security issues

**Comment by abubelinha** [comment]
I understand your python script intended to run on your local machine, but to connect to a remote phpmyadmin web interface. Correct?
In such scenario, you already know phpmyadmin is installed there. Why do you need to verify?
I must be misunderstanding something.

**Comment by davdtheemonk** [comment]
I just thought it as a feature I could add
